CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.0.0 (02/12/2019)
+
+**Common**
+
+* Added a new CMake flag `OLP_SDK_BOOST_THROW_EXCEPTION_EXTERNAL` for a case when SDK needs to compile without exceptions, and the user will provide a custom `boost::throw_exception` function.
+* `olp::client::HttpResponse` now holds a `std::stringstream` instead of `std::string`, this removes expensive copies.
+* CMake minimum version was changed to 3.9.
+* Method `cancel` in CancellationToken is deprecated in favor of a new `Cancel` method.
+* Network logs were improved.
+* Various bug fixes and improvements.
+* Network limit was implemented for all platforms. When the limit reached `Send` method will return `NETWORK_OVERLOAD_ERROR`.
+* Android network client can now be used when Java VM embedded in a native application.
+
+**olp-cpp-sdk-dataservice-read**
+
+* `VersionedLayerClient` and `VolatileLayerClient` are now movable and non-copyable.
+* `TaskContext` and `PendingRequests` classed moved to core component, and are now public.
+* Methods `GetCatalogMetadataVersion` in `CatalogClient` are deprecated, `GetLatestVersion` should be used instead.
+* Deprecated methods were removed.
+* A new parameter version was added to `PrefetchTileRequest`.
```

Comment:
```suggestion
* A new optional parameter for catalog version added to `PrefetchTileRequest` to be used by `VersionedLayerClient::PrefetchTiles()`. If not provided the latest catalog version will be queried from OLP instead.
```

Reply:
If not provided, the latest catalog version is queried from OLP instead.